src/modules/sensors/sensor_params.c

```diff
@@ -31,14 +31,6 @@
  *
  ****************************************************************************/
 
-/**
- * Primary baro ID
- *
- * @category system
- * @group Sensor Calibration
- */
-PARAM_DEFINE_INT32(CAL_BARO_PRIME, 0);
```

Comment:
I guess this was never used, right?

Reply:
Right. We can resurrect easily enough someday if needed.